ev3dev2/motor.py

```diff
@@ -1921,6 +1954,98 @@ def on(self, left_speed, right_speed):
         self.left_motor.run_forever()
         self.right_motor.run_forever()
 
+    def follow_line(self,
+            kp, ki, kd,
+            speed, keep_following_function,
```

Comment:
`predicate`?

Reply:
??

Reply:
It's just not clear from the name what `keep_following_function` is supposed to do... it's really supposed to be a "predicate" with no side-effects. As it is I might interpret the name to mean I have to give a function that keeps following the line. The naming issue is pretty minor though.

Reply:
I am struggling to come up with a better name here :(  The docstring explains the behavior
```
        ``keep_following_function`` is called to determine if we should keep
            following the line or stop
```

I'm open to renaming it if you have suggestions.